src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -6,6 +6,7 @@
     role="listitem"
   >
     <div
+      :aria-selected="item.selected"
```

Comment:
question: should this go on the parent?

Reply:
No, it belongs on the `role="option"` element. Same in KSelectItem